test/e2e_node/remote/node_e2e.go

```diff
@@ -37,7 +37,8 @@ func InitNodeE2ERemote() TestSuite {
 	return &NodeE2ERemote{}
 }
 
-const localCOSMounterPath = "cluster/gce/gci/mounter/mounter"
+const localCOSMounterDir = "cluster/gce/gci/mounter/"
```

Comment:
I'd originally put the kubeconfig file for the node tests in the tarball, and had done this refactoring as part of that. I'll move this refactoring to a separate PR now that it's no longer really related to this one.